### ISSUE TITLE
Raises exception if subreddit doesn't exist

### ISSUE THREAD

**Issue by kshmelkov** [michael-lazar/rtv#127]
When I try to change subreddit, the app failed with the following trace because of non-existing subreddit.

```
Connecting...
Traceback (most recent call last):
  File "/usr/bin/rtv", line 9, in <module>
    load_entry_point('rtv==1.4.1', 'console_scripts', 'rtv')()
  File "/usr/lib/python3.4/site-packages/pkg_resources/__init__.py", line 558, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.4/site-packages/pkg_resources/__init__.py", line 2682, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.4/site-packages/pkg_resources/__init__.py", line 2355, in load
    return self.resolve()
  File "/usr/lib/python3.4/site-packages/pkg_resources/__init__.py", line 2361, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.4/site-packages/rtv/__main__.py", line 138, in <module>
    sys.exit(main())
  File "/usr/lib/python3.4/site-packages/rtv/__main__.py", line 118, in main
    page.loop()
  File "/usr/lib/python3.4/site-packages/rtv/subreddit.py", line 49, in loop
    self.controller.trigger(cmd)
  File "/usr/lib/python3.4/site-packages/rtv/page.py", line 225, in trigger
    return func(self.instance, *args, **kwargs)
  File "/usr/lib/python3.4/site-packages/rtv/subreddit.py", line 92, in prompt_subreddit
    self.refresh_content(name=name)
  File "/usr/lib/python3.4/site-packages/rtv/subreddit.py", line 58, in refresh_content
    self.reddit, name, self.loader)
  File "/usr/lib/python3.4/site-packages/rtv/content.py", line 313, in from_name
    return cls(display_name, submissions, loader)
  File "/usr/lib/python3.4/site-packages/rtv/content.py", line 258, in __init__
    self.get(0)
  File "/usr/lib/python3.4/site-packages/rtv/content.py", line 327, in get
    submission = next(self._submissions)
  File "/usr/lib/python3.4/site-packages/praw-3.1.0-py3.4.egg/praw/__init__.py", line 524, in get_content
    page_data = self.request_json(url, params=params)
  File "/usr/lib/python3.4/site-packages/praw-3.1.0-py3.4.egg/praw/decorators.py", line 173, in wrapped
    return_value = function(reddit_session, *args, **kwargs)
  File "/usr/lib/python3.4/site-packages/praw-3.1.0-py3.4.egg/praw/__init__.py", line 579, in request_json
    retry_on_error=retry_on_error)
  File "/usr/lib/python3.4/site-packages/praw-3.1.0-py3.4.egg/praw/__init__.py", line 423, in _request
    response = handle_redirect()
  File "/usr/lib/python3.4/site-packages/praw-3.1.0-py3.4.egg/praw/__init__.py", line 397, in handle_redirect
    url = _raise_redirect_exceptions(response)
  File "/usr/lib/python3.4/site-packages/praw-3.1.0-py3.4.egg/praw/internal.py", line 178, in _raise_redirect_exceptions
    .format(subreddit))
praw.errors.InvalidSubreddit: `sdfsfasdf` is not a valid subreddit
```

**Comment by Brobin** [comment]
Duplicate of #119, will be fixed in the 1.4.2 release.

**Comment by michael-lazar** [comment]
Alright 1.4.2 is live, please upgrade with

```
sudo pip install -U rtv
```

Sorry about that!
